src/plugins/remote-loader/index.ts

```diff
@@ -5,6 +5,8 @@ import { asPromise } from '../../lib/as-promise'
 import { loadScript } from '../../lib/load-script'
 
 export interface RemotePlugin {
+  /** The name of the action destination. */
```

Comment:
Action destination is one type of remote plugin, this change creates a contract between the two that will likely make it harder for us to evolve the architecture of RemotePlugins

What happens if we want to have edge functions as remote plugins?